health/notifications/alarm-notify.sh.in

```diff
@@ -217,7 +220,7 @@ else
     units="${17}"              # the units of the value
     info="${18}"               # a short description of the alarm
     value_string="${19}"       # friendly value (with units)
-    old_value_string="${20}"   # friendly old value (with units)
+    _="${20}"                  # friendly old value (with units), previously named "old_value_string"
```

Comment:
Is this right? You renamed `old_value_string` with `_`.

Reply:
`old_value_string` is an unused variable so it got renamed to `_` as per https://github.com/koalaman/shellcheck/wiki/SC2034

Reply:
I see the variable documented in https://github.com/netdata/netdata/blob/master/health/notifications/health_alarm_notify.conf#L733 , so some people may have used it in custom notifications. Better put it back, just in case. We can exclude this particular check from the linter.

Reply:
this variable is unused in a script, there is no logic using it.

Reply:
brought it back